lib/wcc/contentful/model_builder.rb

```diff
@@ -60,7 +58,7 @@ def build_model(typedef)
             raise ArgumentError, "These fields do not exist: #{bad_fields}" unless bad_fields.empty?
 
             result =
-              if defined?(context[:preview]) && context[:preview] == ENV['CONTENTFUL_PREVIEW_PASSWORD']
+              if defined?(context[:preview])
```

Comment:
What if they pass in `preview: false`?  It seems like that should cause it to not use preview.